client/package.json

```diff
@@ -4,16 +4,23 @@
   "private": true,
   "dependencies": {
     "photon-colors": "3.3.1",
+    "prop-types": "15.6.2",
     "react": "16.5.1",
     "react-dom": "16.5.1",
     "react-scripts": "1.1.5",
     "watch": "1.0.2"
   },
   "scripts": {
     "start": "react-scripts start",
+    "prebuild": "npm run lint",
     "build": "react-scripts build",
     "watch": "watch 'npm run build' src public",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "./node_modules/.bin/eslint src/*.js"
```

Comment:
I think you can just type `eslint src/*.js` here. Or better, add a `.eslintignore` file containing `node_modules` and `build`, and change it to just `eslint`. 🙂